.github/scripts/label_promoted_commits.py

```diff
@@ -56,9 +73,27 @@ def main():
                 "Accept": "application/vnd.github.v3+json"
             }
             response = requests.get(search_url, headers=headers, params=params)
-            prs = response.json().get("items", [])
-            for pr in prs:
-                refs_match = re.findall(r'Refs (?:#|https.*?)(\d+)', pr["body"])
+            prs = response.json().get("items", [])  # When return empty, it means there are no merged PR only closed
```

Comment:
we are enumerating the pull requests closed by the commits. what do you mean by "no merge PR only closed"?